sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -674,8 +732,8 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
 
   struct PathUpdateResult
   {
-    std::string ETag;
-    std::string LastModified;
+    Azure::Core::Nullable<std::string> ETag;
+    Azure::Core::Nullable<std::string> LastModified;
```

Comment:
Why nullable?

Reply:
ditto

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/1637f4b9ed998c40d9ae3fe054de3382b4769150/sdk/storage/Azure.Storage.Files.DataLake/src/Generated/DataLakeRestClient.cs#L4721

ditto.